site/src/xServices/auth/authXService.ts

```diff
@@ -55,6 +56,12 @@ export const permissionsToCheck = {
     },
     action: "read",
   },
+  [checks.createWorkspaceForUser]: {
+    object: {
+      resource_type: "workspace_creation_for_user",
+    },
+    action: "create",
+  },
```

Comment:
This is unfortunately not a globally check-able thing.

It is:

```json
{
    "object": {
      "resource_type": "workspace",
      "organization_id": "<org_id_here>",
      "owner_id": "<user_id_here>",
    },
    "action": "create",
  }
```

So you have to ask "Can I make a workspace for this user in a given organization".

Reply:
You could do something like `*` for the user id. Essentially choosing a user that does not exist. So "Can I create a workspace for any user in the org"

```json
{
    "object": {
      "resource_type": "workspace",
      "organization_id": "<org_id_here>",
      "owner_id": "*",
    },
    "action": "create",
  }
```